<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PermissionsWidget</class>
 <widget class="QWidget" name="PermissionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User informations</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="5" column="0" colspan="5">
    <widget class="QStackedWidget" name="stack_user_community">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="user_page">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="2" column="0">
        <widget class="UUIDLineEdit" name="lne_user"/>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="btn_add_user">
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <layout class="QGridLayout" name="grid_tbw_user">
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="1">
          <widget class="QToolButton" name="btn_delete_user">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0" rowspan="2">
          <widget class="QTableWidget" name="tbw_user"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="community_page">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="4" column="0">
        <widget class="QLabel" name="lbl_other_community">
         <property name="text">
          <string>Autres communauté</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="UUIDLineEdit" name="lne_community"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="lbl_community_description">
         <property name="text">
          <string>Liste des communautés (publiques et auxquelles j'appartiens)</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="btn_add_community">
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <layout class="QGridLayout" name="grid_tbv_community">
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="1">
          <widget class="QToolButton" name="btn_delete_community">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0" rowspan="2">
          <widget class="QTableView" name="tbv_community"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="10" column="0" colspan="5">
    <widget class="QTableView" name="tbv_offering"/>
   </item>
   <item row="2" column="0" colspan="5">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QRadioButton" name="rbtn_community">
       <property name="text">
        <string>Communautés</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rbtn_user">
       <property name="text">
        <string>Utilisateurs</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="8" column="1" colspan="4">
    <widget class="QgsDateTimeEdit" name="datetime_end_date"/>
   </item>
   <item row="9" column="0" colspan="5">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Sélectionner les services auxquels cette permission donne accès</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Date d'expiration (optionnel)</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="5">
    <widget class="QCheckBox" name="ckb_only_oauth">
     <property name="text">
      <string>Authentification forte requise (nécessite l'utilisation d'une clé OAUTH2)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="lbl_licence">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Licence</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lne_licence"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsDateTimeEdit</class>
   <extends>QDateTimeEdit</extends>
   <header>qgsdatetimeedit.h</header>
  </customwidget>
  <customwidget>
   <class>UUIDLineEdit</class>
   <extends>QLineEdit</extends>
   <header>geoplateforme.toolbelt.lne_uuid</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
